tests/trace-viewer.spec.ts

```diff
@@ -15,9 +15,17 @@
  */
 
 import { enableConfigs, expect, selectConfig, selectTestItem, test, traceViewerInfo } from './utils';
+import { TestItem } from './mock/vscode';
 
 test.skip(({ traceViewerMode }) => !traceViewerMode);
 
+async function waitForTestItemStatus(testItem: TestItem, status: TestItem['status']) {
```

Comment:
It does not sound like we should need it, it is better to make the assertion that follows a waiting assertion.